R/SSplotBiology.R

```diff
@@ -1581,7 +1581,12 @@ SSplotBiology <-
     } # end disable of time-varying growth for multi-season models
 
     # plot time-series of any time-varying quantities
-    if (24 %in% subplots) {
+    if(is.null(MGparmAdj)) {
```

Comment:
should this just be an if{}else{} that is part of the original if statement  below? That way users only get the message if 24 %in% subplots?

Reply:
Yes, good call! I changed to using an if else statement for MGparmAdj